docs/sources/whatsnew/whats-new-in-v9-3.md

```diff
@@ -0,0 +1,81 @@
+---
+_build:
+  list: false
+aliases:
+  - /docs/grafana/latest/guides/whats-new-in-v9-3/
+description: Feature and improvement highlights for Grafana v9.3
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '9.3'
+  - release notes
+title: What's new in Grafana v9.3
+weight: -33
+---
+
+# What’s new in Grafana v9.3 (Beta)
+
+Welcome to Grafana v9.3. If you’d prefer to dig into the details, check out the complete [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+## What's New feature template
+
+Use this template to add your what's new section.
+
+[Generally available | Available in experimental/beta] in Grafana [Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced]
+
+Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
+
+Add a screenshot to this enablement material folder and link to it here.
+
+## Public Dashboards - Audit Table
+
+Available in experimental in Grafana Open Source, Enterprise, Cloud Pro, Cloud Advanced
+
+We have a new feature as part of our Public Dashboard efforts. We have introduced a new menu item under Dashboards → Public Dashboards. This new view is a list of all of the public dashboards in your Grafana instance. From here you can navigate to the underlying dashboard, see if it is enabled, get a quick link out to the public version of the dashboard, or get quick access to the configuration. You will be able to see a dashboard if you have view access, you will be able to navigate to the configuration if you have public dashboard privileges (RBAC “Dashboard Public” or ADMIN basic).
+
+[image public-dashboard-audit-table.png]
+
+## Public Dashboards - Annotations
```

Comment:
@chri2547 Added blurb about annotations suppport

Reply:
super, thank you!